src/domain/exercise.ts

```diff
@@ -0,0 +1,61 @@
+import { Query } from './query';
+
+function isUuidv4(input: string): boolean {
+  const regexExp =
+    /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
```

Comment:
```suggestion
    /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/;
```
Vi burde ikke have brug for hverken global match eller case insensitive. Når REGEX er skrevet som det er.
ellers kunne det være: `/^[0-9A-F]{8}\b-[0-9A-F]{4}\b-[0-9A-F]{4}\b-[0-9A-F]{4}\b-[0-9A-F]{12}$/i`